test/e2e/suite-annotations/redirect.go

```diff
@@ -90,6 +90,39 @@ spec:
 		resp.Header("Location").Equal("https://httpbin.org/sample")
 	})
 
+	ginkgo.It("redirect permanent-redirect in ingress networking/v1", func() {
```

Comment:
We need more test cases to cover `extensions/v1beta1`  and `networking/v1beta1` Ingress.

And should increase the verification of the redirected result.

For example, redirect it to https://httpbin.org/get to check if the result is as expected

Reply:
ok